runner.py

```diff
@@ -0,0 +1,65 @@
+# Text-based adventure game
+
+inventory, wordle_solution, wordle_attempts = [], "magic", 0
+current_room, game_over = "entrance", False
+
+rooms = {
```

Comment:
This would make more sense as a class! Then you could make as many rooms as you'd like and use them the same way, while also being able to define functionality for the rooms.